tests/acceptance/features/apiSharingNg/sharedWithMeSyncDisabled.feature

```diff
@@ -1922,3 +1918,633 @@ Feature: listing sharedWithMe when auto-sync is disabled
         }
       }
       """
+
+    
+  Scenario: user who is also a member of group lists file shared with them from project-space when auto-sync is disabled
```

Comment:
```suggestion
  Scenario: user who is also a member of group lists file shared with them from project-space
```
The feature file name and description at the top of the feature file are clear about the scenarios covered in this file. So, let's not add the text in every description.